clean/louisiana_state_csc_lprr.py

```diff
@@ -9,7 +10,13 @@
 
 
 def standardize_appealed(df):
-    df.loc[:, "appealed"] = df.appealed.str.strip()
+    df.loc[:, "appealed"] = df.appealed.str.strip().fillna("")\
+        .str.replace(r"^(\w+) (\w+)", r'\1 - \2')\
```

Comment:
Please specify the `regex` argument. Otherwise, the output will always contain warnings.

Reply:
Thanks